Tasks/XamariniOS/task.json

```diff
@@ -119,6 +124,7 @@
         },
         {
             "name": "signMethod",
+            "aliases": [ "signOption" ],
```

Comment:
Perhaps "signingOption" would be more like the "signing" parameters that follow. Just a thought. I like prefixes to match :)